packages/appsync-emulator-serverless/__test__/example/serverless.yml

```diff
@@ -0,0 +1,233 @@
+service: appsync-bootstrap-test
+frameworkVersion: '>=1.21.0 <2.0.0'
+provider:
+  name: aws
+  runtime: nodejs8.10
+  stage: dev
+  region: us-east-1
+package:
+  exclude:
+    - node_modules/**
+plugins:
+  - serverless-appsync-plugin
+custom:
+  accountId: 229055845648
```

Comment:
I made the `client-dashboard` repo private because of concerns of making the accountId public, but I just read the following [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html)

```
If you want the URL for your sign-in page to contain your company name (or other friendly identifier) instead of your AWS account ID, you can create an account alias.
```

Reply:
pretty sure we would need exact ID anyhow because IAM uses account id as part of the resources.

Reply:
I started to pass it as a CLI option, but now realize its not really necessary:

`${opt:accountId}`

Reply:
oooh